.env.example

```
@@ -1,19 +1,40 @@
-ANTHROPIC_API_KEY = "your_api_key"
-XAI_API_KEY= "your_api_key"
 NIM_API_KEY="your_api_key"
-EXA_API_KEY="your_api_key"
+
+# Image Generation
 FAL_API_KEY="your_api_key"
+
+# ====================================
+# SEARCH & WEB TOOLS
+# ====================================
+EXA_API_KEY="your_api_key"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)